cmake/Lint.cmake

```diff
@@ -1,21 +1,22 @@
-# Linter
+# Lint
 find_package(PythonInterp 3)
 
-# XXX temporarely disable lint !!!!!!!!!!!
-set(LINT_COMMAND echo lint is disabled!)
```

Comment:
This stuff should definitely get back !!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
actually, I wouldn't mind having linter back :P Let's discuss about it maybe?

Reply:
I DO mind

Reply:
well then let's keep it off... I just thought a short chat about it cannot hurt...

Reply:
It is disabled but on CMakeList.txt.
In my opinion having the switch there make more sense
in this way if you disable the lint CMake don't even need to search for the 
python interpreter.

Reply:
ok then